test/controllers/users/invitations_controller_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class Users::InvitationsControllerTest < ActionController::TestCase
+  test "admin can access invite page" do    
+    user = User.create(:email => "example@example.com", :password => "password", :password_confirmation => "password", :admin => true)
+    sign_in user
+    get :new
```

Comment:
That would probably be helpful. Here it is:

1) Error:
Users::InvitationsControllerTest#test_admin_can_access_invite_page:
NoMethodError: undefined method `name' for nil:NilClass
    test/controllers/users/invitations_controller_test.rb:7:in `block in <class:InvitationsControllerTest>'